addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/handler/BridgeV6Handler.java

```diff
@@ -37,6 +39,8 @@
 public class BridgeV6Handler extends AbstractBridgeHandler implements ISessionState {
     private @NonNullByDefault({}) MilightV6SessionManager session;
     final DateTimeFormatter timeFormat = DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT);
+    private String offlineReasion = "";
```

Comment:
Thanks for fixing the typos!
There seems to be another one here:
offlineReasion -> offlineReason